sphinx4-core/src/main/java/edu/cmu/sphinx/alignment/tokenizer/DecisionTree.java

```diff
@@ -182,19 +182,19 @@ public void dumpDot(PrintWriter out) {
         out.write("rankdir = LR\n");
 
         for (Node n : cart) {
-            out.println("\tnode" + Math.abs(n.hashCode()) + " [ label=\""
+            out.println("\tnode" + (n.hashCode() & Integer.MAX_VALUE) + " [ label=\""
```

Comment:
Previous version was clear - we take absolute of the hash code. The new version is not clear at all.